pkg/console/subresource/deployment/deployment.go

```diff
@@ -421,3 +434,34 @@ func customLogoVolume() volumeConfig {
 		path:        "/var/logo/",
 		isConfigMap: true}
 }
+
+// redeployRSVHash is a hash of the resource versions of important
+// config resources. If any of these resources changes, the hash should
+// change, causing a rollout of the deployment.
+// The operator config is omitted as it would cause redeploy loops (as the
+// status is updated frequently), but user driven operator config changes
+// will already cause redeploy.
+func redeployRSVHash(rsvList []string) string {
```

Comment:
Are these functions taken from somewhere ? If so would be good to reference the source, so we can tract it for changes

Reply:
They are a nod to something done inline in the [authentication-operator here](https://github.com/openshift/cluster-authentication-operator/blob/33f47a4ca93995a49b7429cf953f54895eccba32/pkg/operator2/deployment.go#L96), but not a copy over of exact functions.

Reply:
great ! just wanted to check, in case we need to update